src/get-queries-for-element.js

```diff
@@ -1,7 +1,22 @@
-import * as queries from './queries'
+import * as defaultQueries from './queries'
 
-function getQueriesForElement(element) {
-  return Object.entries(queries).reduce((helpers, [key, fn]) => {
+/**
+ * @typedef {{[key: string]: Function}} FuncMap
+ */
+
+/**
+ * @param {HTMLElement} element container
+ * @param {FuncMap|FuncMap[]} queries object of functions, or array of objects
+ * @returns {FuncMap} returns object of functions bound to container
+ */
+function getQueriesForElement(element, queries = defaultQueries) {
```

Comment:
Note that passing custom queries _overrides_ the default instead of merging. If you want to merge you need to import `queries` and add it to the array, too.

Open to changing this but I think it makes sense; wrappers like react-testing-library's `render` can merge queries by default on top of this API if they prefer.